experimental/origin-dapp2/src/pages/create-listing/mutations/CreateListing.js

```diff
@@ -66,7 +66,8 @@ class CreateListing extends Component {
           category: listing.category,
           subCategory: listing.subCategory,
           media: listing.media,
-          unitsTotal: Number(listing.quantity)
+          unitsTotal: Number(listing.quantity),
+          isMultiUnit: Number(listing.quantity) > 1
```

Comment:
This will need to be eventually updated to `type === 'unit' && Number(listing.quantity) > 1` but we don't have that `type` concept here yet.

Reply:
I also did this instead of "asking" for a checkbox as it seemed like a better UX.